versions/latest/modules/en/pages/rancher-admin/scc-registration.adoc

```diff
@@ -0,0 +1,107 @@
+= SUSE Customer Center Registration
+:revdate: 2025-08-19
+:page-revdate: {revdate}
+
+Since Rancher v2.12.1, you should now register your SUSE Rancher Manager Prime Suite with https://scc.suse.com/home[SUSE Customer Center (SCC)] from the Rancher UI or through Helm. This will enable you to see your registered systems in SCC under your subscription. 
```

Comment:
`you should` - is registration a recommendation or mandatory?

Reply:
@LucasSaintarbor I'm going to add an approval since the phrasing change by https://github.com/rancher/rancher-product-docs/pull/404/commits/d7eca854c255107f4654a6f3e424831f4bd1773c skirts around this, but we should still find an answer because it's likely something that will be asked.